src/services/completions.ts

```diff
@@ -637,7 +638,7 @@ namespace ts.Completions {
             if (type) {
                 // Filter private properties
                 for (const symbol of type.getApparentProperties()) {
-                    if (typeChecker.isValidPropertyAccess(<PropertyAccessExpression>(node.parent), symbol.name)) {
+                    if (typeChecker.isValidPropertyAccess(<PropertyAccessExpression>(node.parent), symbol.getName())) {

```

Comment:
Previously `getName()` was identical to `name` and we use that pattern elsewhere.
Also, this is unescaping the name, and then escaping it again in the checker.